app/views/users/_main_feed.html.erb

```diff
@@ -8,11 +8,11 @@
 <% end %>
 <% @stories.each_with_index do |story, i| %>
   <%= render "articles/single_story", story: story %>
-  <% if i == 0 && @comments.any? %>
+  <% if i == 0 && @comments.present? %>
```

Comment:
`users/comment_section` will definitely load comments, so we can preload them here, this, combined with the changes inside the partial, saves one or two queries